src/_includes/graphql/customer-orders-output.md

```diff
@@ -0,0 +1,343 @@
+
+The `orders` attribute defines a filter that returns details about one or more of the logged-in customer's previous orders. It takes the following attributes as input:
+
+Attribute | Data type | Description
+--- | --- | ---
+`filter` | CustomerOrdersFilterInput | Defines the criteria to search for. If no filter is specified, the query returns the customer's first order
+`currentPage` | Int | Specifies which page of results to return. The default value is 1
+`pageSize` | Int | Specifies the maximum number of results to return at once. The default value is 20
+
+The `customers` query returns a [`CustomerOrders`](#customerOrders) object.
+
+The following example shows a fully-defined `orders` attribute:
```

Comment:
It's not clear what you're trying to show with this example, maybe just a full/basic example query to get customer orders